runtime/js/99_main.js

```diff
@@ -94,6 +94,9 @@ import { bootstrap as bootstrapOtel } from "ext:deno_telemetry/telemetry.ts";
 if (Symbol.metadata) {
   throw "V8 supports Symbol.metadata now, no need to shim it";
 }
+
+/// REMOVE
```

Comment:
What does this mean?

Reply:
Accidental left-over from when the cache-busting didn't work properly.